js/viz.js

```diff
@@ -45,22 +49,44 @@ const viz = {
       simulation.nodes(this.nodes);
     }
 
+    this.registerSimulationForces(simulation);
+
+    simulation.alphaTarget(1);
+    for (let i = 0; i < 10; i++) {
+      simulation.tick();
+    }
```

Comment:
manually updating `tick` to avoid the node collisions. @jonathanKingston let me know what you think.